proposals/build-provenance-and-code-signing-for-homebrew.md

```diff
@@ -110,3 +110,55 @@ For formulae with verifiable source artifacts, the signed bottle attestation des
 ### macOS executable code signing
 
 Because this proposal only concerns bottles and not their individual executable members, it should have no effect on Homebrew's current use or any future uses of "ad-hoc" executable signatures (which are produced not for authenticity, but because macOS on Apple Silicon requires all executables to be signed).
+
+## Threat model
+
+### Scope
+
+The scope of this proposal is *authenticated build provenance* for Homebrew bottles.
```

Comment:
Should we mention somewhere that source compromise is out of scope?

Reply:
Yeah, I'll include that in this section.

Reply:
Done.